language-support/ts/daml-react/README.md

```diff
@@ -63,7 +63,8 @@ const [choiceReturnValue, events] = await ledger.exercise(ContractChoice, contra
 `useQuery`
 ----------
 `useQuery` returns the contracts matching a given query. The query matches for a given contract
-template and specified field values of the contracts of that template.
+template and specified field values of the contracts of that template. If the query is omitted, all
+visible contracts of the given template are returned.
```

Comment:
I think it would help to have a code example for this as well. And down for `useStreamQuery` too.